frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/RebaseCurrentBranchOntoParentAction.java

```diff
@@ -21,11 +22,16 @@ protected void onUpdate(AnActionEvent anActionEvent) {
       return;
     }
 
-    var isNonRootBranch = getCurrentBranchNameIfManaged(anActionEvent)
-        .flatMap(bn -> getManagedBranchByName(anActionEvent, bn))
-        .map(b -> b.isNonRoot())
+    var currentBranch = getCurrentBranchNameIfManaged(anActionEvent)
+        .flatMap(bn -> getManagedBranchByName(anActionEvent, bn));
+
+    var isNonRootBranch = currentBranch.map(b -> b.isNonRoot()).getOrElse(false);
+
+    @SuppressWarnings("confirmedbranchitem") var isNotMerged = currentBranch
```

Comment:
Seems to subtype checker not working with filter

Reply:
You can also change type of `var currentBranch` from `Option<X>` to `X` by `.getOrNull()`... but that's a detail, not insisting

Reply:
Nit: if the suppression stays, maybe add the prefix for clarity: `subtyping:confirmedbranchitem`? (or whatever the correct prefix happens to be)